en/tidb-operator-overview.md

```diff
@@ -14,22 +14,33 @@ category: reference
+* `TidbInitializer` describes the desired initialization Job of the TiDB cluster.
+* `Backup` describes the desired backup of the TiDB cluster.
+* `Restore` describes the desired restoration of the TiDB cluster.
+* `BackupSchedule` describes the scheduled backup of the TiDB cluster.
+* `TidbClusterAutoScaler` describes the automatic scaling of the TiDB cluster.
+
+The following components are responsible for the orchestration and scheduling logic in a TiDB cluster:
+
+* `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state and complete the corresponding control logic according to other CR;
+* `tidb-scheduler` is a Kubernetes scheduler extension that injects the TiDB specific scheduling policies to the Kubernetes scheduler;
+* `tidb-admission-webhook` is a dynamic admission controller in Kubernetes, which completes the modification, verification, operation and maintenance of Pod, StatefulSet and other related resources.
 
 In addition, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
 
-![TiDB Operator Control Flow](/media/tidb-operator-control-flow.png)
+![TiDB Operator Control Flow](/media/tidb-operator-control-flow-1.1.png)
 
-The diagram above is the analysis of the control flow of TiDB Operator. Because TiDB clusters also need components such as monitoring, initialization, scheduled backup, Binlog and so on, TiDB Operator encapsulates the definition of these components in the Helm chart. The overall control process is as follows:
+The diagram above is the analysis of the control flow of TiDB Operator. Starting from TiDB Operator v1.1, the TiDB cluster, monitoring, initialization, backup, and other components are deployed and managed using CR. The overall control flow is described as follows:
 
-1. The user creates a `TidbCluster` object and a corresponding series of Kubernetes-native objects through Helm, such as a `CronJob` that performs scheduled backups;
-2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet` and `Service` objects of PD, TiKV, and TiDB based on the actual state of the cluster;
-3. Kubernetes' native controller creates, updates, and deletes the corresponding `Pod` based on objects such as `StatefulSet`, `Deployment`, and `CronJob`;
+1. The user creates a `TidbCluster` object and other CR objects through kubectl, such as `TidbMonitor`;
+2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet`, `Deployment` and `Service`, and other objects of PD, TiKV, TiDB, Monitor or other components based on the actual state of the cluster;
```

Comment:
```suggestion
2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet`, `Deployment`, `Service`, and other objects of PD, TiKV, TiDB, Monitor or other components based on the actual state of the cluster;
```

Reply:
"the `StatefulSet`, `Deployment`, `Service`, and other objects" or "the `StatefulSet`, `Deployment`, `Service` and other objects"?

Reply:
I think the former is right. When you describe a series of items, put a comma before the final `and` or `or`. You can refer to the [Google Style Guide | commas](https://developers.google.cn/style/commas#serial-commas_1) 😀

Reply:
OK.